### PR TITLE
:recycle: Apply sourcery suggestions

### PR DESCRIPTION
**Pull request by Jerry-py**
## Description

Stuff

## Checklist

- [ ] - All code changes have been tested, if any
- [ ] - The documentation has been updated accordingly for any code changes
- [ ] - This PR fixes an already existing issue
- [ ] - This PR adds a new feature (new functions, methods, classes, parameters, etc.)
- [ ] - This PR fixes a bug
- [x] - This PR is a breaking change (removes/renames functions, parameters etc.)
- [ ] - This PR is NOT a code change (changes Documentation files, README.md, CONTRIBUTING.md etc.)

### CODE REVIEW

File: EpikCord/abstract.py [hunk 1]
```diff
@@ -77,7 +77,7 @@ async def __aexit__(self):
 
 class Messageable:
     def __init__(self, client: Client, channel_id: int):
-        self.id: int = int(channel_id)
+        self.id: int = channel_id
```
Comment (TheUntraceable) [inline EpikCord/abstract.py:80]: I have a feeling that internally something is going to go wrong.

**Review by Sigmanificient — APPROVED** [review]
lgtm, refer to @TheUntraceable comment

**Comment by Sigmanificient** [comment]
Wait is the pr meant to be for rewrite or master?

**Comment by TheUntraceable** [comment]
> Wait is the pr meant to be for rewrite or master?

Rewrite. I changed it.

**Comment by Jerry-py** [comment]
Wrong branch from my side as well sorry

**Pull request closed without merging**